packages/web/src/components/organisms/TriggersFormItems/ActionFormItems.tsx

```diff
@@ -66,7 +72,7 @@ const ActionFormItems = () => {
             name="testNameSelector"
             rules={[required]}
           >
-            <ResourceTriggerSelect />
+            <ResourceTriggerSelect actionValue={actionValue} />
```

Comment:
Probably it would be better to move the responsibility - `ActionFormItems` understands the action value, but `ResourceTriggerSelect` shouldn't.

Instead, you may add props `tests`/`testSuites` (or `allowTests`/`allowTestSuites`) to the `ResourceTriggerSelect`:
```suggestion
            <ResourceTriggerSelect allowTests={actionValue === 'run test'} allowTestSuites={actionValue === 'run testsuite'} />
```

This way:
* `ResourceTriggerSelect` may easily skip unnecessary resources query
* `ResourceTriggerSelect` may show any set of resources, as per parent's expectation